content/sensu-go/5.16/installation/install-sensu.md

```diff
@@ -85,6 +85,8 @@ sudo yum install sensu-go-backend
 
 ### 2. Initialize
 
+_**NOTE**: If you are using Docker, skip this step and continue with [3. Configure and start][36]. The `sensu-backend init` command is not implemented for Docker._
+
```

Comment:
I added this note based on https://sensu.slack.com/archives/C60EEQFH8/p1576190537067300

Reply:
Maybe: "The 'sensu-backend init` command is automatically ran for you with the default username and password of `admin` and `P@ssw0rd!`."

Saying "The `sensu-backend init` command is not implemented for Docker." is incorrect as the binary inside the container will have that comman available, you just don't need to run it to get the admin user provisioned as we are taking care of that for you in 5.16.

Reply:
^ updated @rgeniesse wdyt?